helm/kong-app/templates/admission-webhook.yaml

```diff
@@ -17,6 +17,8 @@
 {{- if $certSecret }}
 {{- $certCert = (b64dec (get $certSecret.data "tls.crt")) -}}
 {{- $certKey = (b64dec (get $certSecret.data "tls.key")) -}}
+{{- end }}
+{{- if $caSecret }}
```

Comment:
The problem is because there are separate variables for the CA and the cert.

https://github.com/giantswarm/kong-app/blob/d2afb4ab8a6ad8ae9fdad75be2707c1982eb970b/helm/kong-app/templates/admission-webhook.yaml#L15

So if the CA secret is new but the cert isn't the data will be missing when the template is rendered.